config/hysr_one_ball_default_sim.json

```diff
@@ -14,7 +14,7 @@
         "min":[0.0,-1.0,0.17],
         "max":[1.6,3.5,1.5]
     },
-    "pam_config_file":"/opt/mpi-is/pam_interface/pam_sim.json",
+    "pam_config_file":"/opt/mpi-is/pam_interface/pamy2/pam.json",
```

Comment:
I think the same change is necessary in `hysr_demos.json` and probably in the example config in the README.